feat: add internal loading for bridge tx list

LuizAsFight opened the pull request.

matt-user reviewed (commented):
could you add the loader versions of the components to the storybook?

LuizAsFight commented:
yes @matt-user thanks for the catch. I added the loading version for both `BridgeTxOverview` and `BridgeTxItem`. also fixed asset not showing correctly in `BridgeTxOverview`story
